lib/index.js

```diff
@@ -104,7 +104,8 @@ function clean(obj) {
   for (var k in obj) {
     if (obj.hasOwnProperty(k)) {
       var value = obj[k];
-      if (value === null) continue;
+      // don't send null or undefined values
```

Comment:
As a general rule: Comments should only ever describe _what_ the code is doing when the code is very complex. Figuring out what a line of code is doing without help is usually pretty easy. Figuring out the context behind that code (the _why_) is usually difficult or impossible; your comments should aspire to answer this question